ui/src/components/Header/Header.js

```diff
@@ -0,0 +1,23 @@
+import Image from '../Global/Image/Image';
+import Button from '../Global/Button/Button';
```

Comment:
We should install the module package (apologies I don't know the technical name for that) basically it helps in avoiding repetitive path

Reply:
Unable to configure that here. Do you have any reference I can look at?